src/test/java/integration/helper/TestHelper.java

```diff
@@ -131,6 +131,12 @@ public void deleteShipment(Shipment shipment) throws Exception {
             log.debug(e.getMessage());
         }
     }
+    
+    public JSONObject toJsonClientUuidObject(Client client) {
```

Comment:
Plz, rename to clientToJson without specifying "uuid" in the name, cuz we don't use it for other purpose.